tests/components/dyson/test_sensor.py

```diff
@@ -191,7 +197,7 @@ def test_dyson_humidity_standby_monitoring(self):
         sensor.entity_id = "sensor.dyson_1"
         assert not sensor.should_poll
         assert sensor.state == STATE_OFF
-        assert sensor.unit_of_measurement == "%"
+        assert sensor.unit_of_measurement == UNIT_PERCENTAGE
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5093397)